### PR TITLE
Add debug printing for ll

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Partly solves #92

### CODE REVIEW

File: src/libreset/ll.h [hunk 1]
```diff
@@ -33,6 +33,7 @@
 #define __LL_H__
 
 #include "util/attributes.h"
+#include "util/debug.h"
```
Comment (neithernut) [inline src/libreset/ll.h]: Not needed.

Comment (matthiasbeyer) [inline src/libreset/ll.h]: Why not? Sure this is needed for the `dbg()` macro!

Comment (matthiasbeyer) [inline src/libreset/ll.h]: Ah, it was the header,...

**Comment by neithernut** [comment]
ACK.

**Comment by matthiasbeyer** [comment]
Waiting on CI after rebase.

**Pull request merged**